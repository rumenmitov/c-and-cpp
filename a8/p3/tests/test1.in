a
3
a
5
a
7
p
q
