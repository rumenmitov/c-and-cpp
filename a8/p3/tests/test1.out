add int: Putting 3 into queue
1 items in queue
Type a to add, d to delete, p to print, q to quit:
add int: Putting 5 into queue
2 items in queue
Type a to add, d to delete, p to print, q to quit:
add int: Putting 7 into queue
3 items in queue
Type a to add, d to delete, p to print, q to quit:
content of the queue: 3 5 7 
3 items in queue
Type a to add, d to delete, p to print, q to quit:
Bye.
